src/GeneralStateTestsFiller/stZeroKnowledge/pairingTestFiller.json

```diff
@@ -130,7 +130,7 @@
 		"[1] vk.B + proof.B + negateG1(proof.Bp) + P2",
 		"[2] proof.C + vk.C + negateG1(proof.Cp) + P2",
 		"[3] proof.K + vk.gamma + negateG1(addG1(vkx, addG1(proof.A, proof.C))) + vk.gammaBeta2 + negateG1(vk.gammaBeta1) + proof.B",
-		"[4] addG1(vkx, proof.A) + proof.B + negateG1(proof.H) + vk.Z +	negateG1(proof.C) + P2",
+		"[4] addG1(vkx, proof.A) + proof.B + negateG1(proof.H) + vk.Z + negateG1(proof.C) + P2",
 		"[5] proof.A + vk.A + proof.Ap + P2"
 	    ],
             "data" : [
```

Comment:
technically duplicate keys are valid JSON. I had to look it up: "The names within an object SHOULD be unique." So duplicate keys are not recommended but not forbidden.

Luckily the schema validator we're using only validates the last key/value when there are duplicates.